### PR TITLE
chore: moving to opa-la services and remove opa+envoy from service

### PR DESCRIPTION
**Pull request by ronenkapelian**
- remove internal auth process via opa+envoy inside nginx pod.
- connecting the nginx making auth with opa-la

### CODE REVIEW

**Review by alebinson — CHANGES_REQUESTED** [review]
Should wait till @AnguIar final solution with code sharing pronciples

**Review by asafMasa — APPROVED** [review]

File: helm/raster/config/nginx.conf [hunk 1]
```diff
@@ -33,9 +31,30 @@ server {
...
+      internal;
+
+      js_content auth.opaAuth;
+    }
+
+    location = /opa {
+      internal;
+
+      proxy_set_header Content-Type application/json;
+      proxy_pass {{ .Values.rasterCommon.authentication.opa.opaUrl }};
```
Comment (netanelC) [inline helm/raster/config/nginx.conf:58]: opaUrl field does not exist in values.yaml

File: helm/raster/templates/nginx-deployment.yaml [hunk 2]
```diff
@@ -79,10 +67,16 @@ spec:
           image: {{ $cloudProviderDockerRegistryUrl }}{{ .repository }}:{{ .tag }}
           {{- end }}
           imagePullPolicy: {{ .Values.imagePullPolicy }}
+          env:
+          - name: DOMAIN
```
Comment (netanelC) [inline helm/raster/templates/nginx-deployment.yaml:71]: Could you let me know why you added the domain? in nginx.conf file, you already set the domain variable directly from the values.yaml and not from env.

**Review by netanelC — CHANGES_REQUESTED** [review]
Please see my review :)

File: helm/raster/templates/nginx-service.yaml [hunk 3]
```diff
@@ -27,14 +27,8 @@ spec:
     - port: {{ .Values.nginx.port }}
       targetPort: {{ .Values.nginx.targetPort }}
       protocol: TCP
-      name: internal
+      name: nginx
       # internal
```
Comment (netanelC) [inline helm/raster/templates/nginx-service.yaml:31]: Irrelevant anymore

**Pull request closed without merging**